### PR TITLE
Add django_auth_ldap

### PR DESCRIPTION
**Pull request by szok**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,6 @@
 FROM zefciu/dnsaas-base
 MAINTAINER Pylabs <pylabs@allegro.pl>
+RUN apt-get update && apt-get install -y libsasl2-dev python-dev libldap2-dev libssl-dev
```
Comment (mkurek) [inline Dockerfile]: isn't `python-dev` python2? do we need it?

**Review by mkurek — COMMENTED** [review]
Did you consider making separated package with this code and use it both in ralph and dnsaas?

File: Dockerfile-integration [hunk 2]
```diff
@@ -1,5 +1,6 @@
 FROM zefciu/python-base
 MAINTAINER Pylabs <pylabs@allegro.pl>
+RUN apt-get update && apt-get install -y libsasl2-dev python-dev libldap2-dev libssl-dev
```
Comment (xliiv) [inline Dockerfile-integration]: https://github.com/allegro/django-powerdns-dnssec/pull/149/files?diff=split#r79801742

File: Dockerfile (hunk 1 above)
Comment (szok) [reply Dockerfile]: Done :)

**Comment by mkurek** [comment]
Newest version: instead of making separate package, lets try to make pull requests to django-auth-ldap (https://bitbucket.org/psagers/django-auth-ldap/wiki/Home)

**Comment by mzdeb** [comment]
This project is deprecated.

**Pull request closed without merging**